test/robot/build/build.proto

```diff
@@ -53,8 +53,8 @@ message Information {
   string tag = 4;
   // Description is a user assigned description of the build.
   string description = 5;
-  // Builder is the device information for the machine that made the build.
-  device.Instance builder = 6;
+  // BuilderAbi is the device ABI for the machine that made the build.
+  device.ABI builder_abi = 6;
```

Comment:
This change is not really necessary - you can just leave the other fields blank. Also, I just noticed that the ToolSet itself has the ABI, so it is now duplicated.

Instead, for Kokoro builds, we should just leave the builder empty (except maybe put "Kokoro" into the name field) and then use the ABI from the ToolSet to match at test run time.

Reply:
The ToolSet populates its ABI based on this ABI when it is created on the server, so we still need that at the very least. I will leave the Instance and put Kokoro in the name anyways.

Reply:
Updated.